code/ZAS/Zone.dm

```diff
@@ -199,6 +204,42 @@ Class Procs:
 	//for(var/turf/T in unsimulated_contents)
 //		to_chat(M, "[T] at ([T.x],[T.y])")
 
+
+/zone
```

Comment:
zone is already defined at the top of this file, why don't you put your var there instead of making a new definition here? it is far more maintainable

Reply:
I wanted to separate the "event-only" variables from the rest of the /zone vars so that it could be clear that ice_tiles_list and whatever else may be added in the future is only used in these procs. Do you think that's a bad idea? I guess I wanted to emulate C++ subclass design, so this would sort of be the equivalent of a "class Zone::EventData".

Reply:
Yeah, we don't do it that way here, but it's an easy fix [caveat: you may see some extremely ancient things like this in the code in places from Bay era]